frontend/pages/shopping-lists/_id.vue

```diff
@@ -58,7 +58,15 @@
       <div v-else>
         <div v-for="(value, key) in itemsByLabel" :key="key" class="mb-6">
           <div class="text-left">
-            <v-btn :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#959595'">{{ key }}</v-btn>
+            <v-btn
+              :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#959595'"
+              :style="{
+                'color': getTextColor(getLabelColor(value[0])),
+                'letter-spacing': 'normal',
```

Comment:
The only thing I'm a bit unsure about is that it feels slightly cramped, though that might just be because it's such a contrast to how it was before.

Let's merge it as is, and if it ends up feeling too tight in the long run, we can adjust the spacing later.

Reply:
It's funny, I felt the opposite probably because that's how it appears elsewhere on labels, lol. Yeah we can always change it